Kinvey/Kinvey/DataStore.swift

```diff
@@ -214,6 +227,7 @@ open class DataStore<T: Persistable> where T: NSObject {
     @discardableResult
     open func remove(_ persistable: T, writePolicy: WritePolicy? = nil, completionHandler: IntCompletionHandler?) throws -> Request {
         guard let id = persistable.entityId else {
+            log.severe("Object Id is missing")
```

Comment:
Right, since the user can catch the exception, it should be `log.error`. just changed!